rllib/env/multi_agent_env_runner.py

```diff
@@ -259,7 +263,7 @@ def _sample_timesteps(
                 to_env = {
                     Columns.ACTIONS: [
                         {
-                            aid: self.env.get_action_space(aid).sample()
+                            aid: self.env.unwrapped.get_action_space(aid).sample()
```

Comment:
This is only RLlib-internally b/c the vector env is wrapped to return info-lists (rather than dicts).
I think we still have to unify the way the user sees these envs in their callbacks: Always as-is (vector wrapper) or always as unwrapped (pure gym.vector.Env). Both approaches have advantages and disadvantages.